certified-connectors/RecordedFutureV2/apiDefinition.swagger.json

```diff
@@ -1,913 +1,1530 @@
+                  "format": "string"
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/playbook-alert/{id}": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Default",
+            "schema": {
+              "$ref": "#/definitions/PlaybookAlertLookup"
+            }
+          }
+        },
+        "summary": "Get Playbook Alert by ID",
+        "operationId": "Playbook_Alert_Lookup",
+        "description": "Get the alert details of a playbook alert",
+        "x-ms-visibility": "advanced",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Playbook Alert ID",
+            "x-ms-summary": "Playbook Alert ID",
+            "x-ms-visibility": "important"
+          }
+        ]
+      }
+    },
+    "/detection-rules/search": {
+      "post": {
+        "summary": "Search Detection Rules",
+        "operationId": "Detection_Rule_Search",
+        "description": "Get detection rules matching a search filter",
+        "x-ms-visibility": "advanced",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "body",
+            "in": "body",
+            "required": false,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "types": {
+                  "type": "array",
+                  "description": "List of detection rule types to include in the response",
+                  "items": {
+                    "type": "string",
+                    "enum": [
+                      "sigma",
+                      "yara",
+                      "snort"
+                    ]
+                  }
+                },
+                "entities": {
+                  "type": "array",
+                  "description": "List of entities that the detection rules must be related to",
+                  "items": {
+                    "type": "string",
+                    "enum": [
+                      "CVE-2000-01",
+                      "mitre:TA0001"
+                    ]
+                  }
+                },
+                "created": {
+                  "type": "object",
+                  "properties": {
+                    "before": {
+                      "description": "Limit the response to detection rules created before this date",
+                      "type": "string",
+                      "format": "date-time",
+                      "enum": [
+                        "2023-06-01T18:00:00Z"
+                      ]
+                    },
+                    "after": {
+                      "description": "Limit the response to detection rules created after this date",
+                      "type": "string",
```

Comment:
@lindarecfut The type of the enum value is 'Date' but it should be 'String' as specified in the schema.